packages/components/src/components/hds/code-editor/description.ts

```diff
@@ -3,18 +3,23 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
-import TemplateOnlyComponent from '@ember/component/template-only';
+import Component from '@glimmer/component';
 
 import type { HdsTextBodySignature } from '../text/body';
 
+type HdsCodeEditorDescriptionElement = HdsTextBodySignature['Element'];
 export interface HdsCodeEditorDescriptionSignature {
+  Args: {
+    editorId: string;
+    tag?: HdsTextBodySignature['Args']['tag'];
```

Comment:
Suggestion: remove the tag argument, it isn't being used in the hbs file or this one.